NebulaPatcher/Patches/Dynamic/PlanetModelingManager_Patch.cs

```diff
@@ -5,6 +5,13 @@
 using System.Collections.Generic;
 using System.Linq;
 
+using System.Security;
+using System.Security.Permissions;
+using UnityEngine;
+
+[module: UnverifiableCode]
+[assembly: SecurityPermission(SecurityAction.RequestMinimum, SkipVerification = true)]
+
```

Comment:
I don't think that you still need `System.Security` and `SecurityPermission` since you now use reflection ?